### PR TITLE
Adding descending sort (Z-A) in a bazaar plugin 

### PR DESCRIPTION
**Pull request by AmbrishRamachandiran**
## Hey, I just made a Pull Request!

By default bazaar plugin project have sorting 

Default sort

Adding  descending sort (Z-A) will help some users and improve user experience

Z-A sort

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by AmbrishRamachandiran** [comment]
I've been enjoying using backstage plugins and I'd like to request the addition of Z-A sorting for the content. While the current A-Z and recent sorting options are helpful, having a Z-A option would be a great addition for these reasons:

1. **Enhanced Navigation:** Z-A sorting can make it easier to find older content, which can be valuable in certain use cases.

2. **Alignment with Industry Standards:** Many repositories in our field offer Z-A sorting as a standard feature. This addition would align with common industry practices.

I understand that development priorities can be demanding, but I believe this feature would be well-received. Please consider the possibility of adding Z-A sorting.

Thanks for your time and consideration! 👍

File: plugins/bazaar/src/util/sortMethods.ts [hunk 1]
```diff
@@ -35,6 +35,15 @@ export const sortByTitle = (a: BazaarProject, b: BazaarProject) => {
   return 0;
 };
 
+export const sortByDecendingTitle = (a: BazaarProject, b: BazaarProject) => {
+  if (a.title < b.title) {
```
Comment (vinzscam) [inline plugins/bazaar/src/util/sortMethods.ts:39]: comparing strings using `<` or `>` operators might produce wrong results, I would suggest another approach

Comment (AmbrishRamachandiran) [reply plugins/bazaar/src/util/sortMethods.ts]: @vinzscam please review have modified the logic

Comment (vinzscam) [reply plugins/bazaar/src/util/sortMethods.ts]: nice, thank you for fixing the other method as well

File: plugins/bazaar/src/util/sortMethods.ts [hunk 2]
```diff
@@ -27,12 +27,11 @@ export const sortByDate = (a: BazaarProject, b: BazaarProject): number => {
...
-  if (a.title < b.title) {
-    return -1;
-  } else if (a.title > b.title) {
-    return 1;
-  }
-  return 0;
+  return a.title.localeCompare(b.title);
+};
+
+export const sortByDecendingTitle = (a: BazaarProject, b: BazaarProject) => {
```
Comment (vinzscam) [inline plugins/bazaar/src/util/sortMethods.ts:33]:
```suggestion
export const sortByDescendingTitle = (a: BazaarProject, b: BazaarProject) => {
```

**Review by freben — APPROVED** [review]

**Pull request merged**